src/index.js

```diff
@@ -0,0 +1,216 @@
+const airbnb = require('neutrino-preset-airbnb-base');
+const banner = require('neutrino-middleware-banner');
+const loaderMerge = require('neutrino-middleware-loader-merge');
+const react = require('neutrino-preset-react');
+const { merge } = require('neutrino-middleware-compile-loader');
+const nodeExternals = require('webpack-node-externals');
+const { readdirSync } = require('fs');
+const { basename, join } = require('path');
+
+const MODULES = join(__dirname, 'node_modules'); // eslint-disable-line padding-line-between-statements
+
+module.exports = (neutrino, options = {}) => {
+  if (
+    (options.components || options.library) &&
+    neutrino.options.output.endsWith('build')
+  ) {
+    neutrino.options.output = 'lib'; // eslint-disable-line no-param-reassign
+
+    try {
+      const pkg = require(join(neutrino.options.root, 'package.json')); // eslint-disable-line
+      const hasSourceMap =
+        (pkg.dependencies && 'source-map-support' in pkg.dependencies) ||
+        (pkg.devDependencies && 'source-map-support' in pkg.devDependencies);
+
+      if (hasSourceMap) {
+        neutrino.use(banner);
+      }
+    } catch (ex) {} // eslint-disable-line
+  }
+
+  neutrino.config.resolve.modules.add(MODULES);
+  neutrino.config.resolveLoader.modules.add(MODULES);
+
+  neutrino.use(airbnb, {
+    eslint: {
+      plugins: ['eslint-plugin-react', 'eslint-plugin-prettier'],
+      baseConfig: {
+        extends: [
+          'plugin:react/recommended',
+          'eslint-config-prettier',
+          'eslint-config-prettier/react'
+        ]
+      },
+      rules: {
+        // There are external dependencies we sometimes can't control, and
+        // renaming their variable to lowercase would not be idiomatic
+        'babel/new-cap': 'off',
```

Comment:
IMO we should either upstream these to Neutrino, here:
https://github.com/mozilla-neutrino/neutrino-dev/blob/v7.1.0/packages/neutrino-preset-airbnb-base/index.js#L11-L20

...or else stick to defaults.